_posts/2019-02-22-indico-2-2-news.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Introducing Indico 2.2 - featuring completely new room booking system!"
+date:   2019-02-21 10:00
```

Comment:
I'd bump the date to today (and align all or none of the values? :D)

Reply:
we have this strange alignment in all the posts, I'll fix it